.travis.yml

```diff
@@ -23,3 +23,9 @@ jobs:
   #     - dapp testnet &
   #     - ./bin/deploy.sh
   #     - npm run test
+  - stage: Publish patch
+    if: "(NOT type IN (pull_request)) AND (branch = master)"
+    script:
+      - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
```

Comment:
Do we really want to redirect stderr?

Reply:
I copied this from the way it's done in axis. Travis CI isn't setup yet (I don't have access, have requested this from Lucas). So I'd suggest we keep it like this for now and I revise/check this once Travis CI is set up.

Reply:
Sounds good, thanks!